ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/platform.j2

```diff
@@ -4,16 +4,23 @@
 {%     if platform.trident.forwarding_table_partition is arista.avd.defined %}
 platform trident forwarding-table partition {{ platform.trident.forwarding_table_partition }}
 {%     endif %}
-{%     if platform.sand.lag.hardware_only is arista.avd.defined(true) %}
+{%     if platform.sand is arista.avd.defined %}
+{%         for qos_map in platform.sand.qos_maps | arista.avd.natural_sort('traffic_class') %}
+{%             if qos_map.traffic_class is arista.avd.defined and qos_map.to_network_qos is arista.avd.defined %}
+platform sand qos map traffic-class {{ qos_map.traffic_class }} to network-qos {{ qos_map.to_network_qos }}
```

Comment:
@carlbuchmann let me know if the order of command is proper.

Reply:
Yes here's an example: 

```eos
platform sand qos map traffic-class 0 to network-qos 0
platform sand qos map traffic-class 1 to network-qos 8
platform sand qos map traffic-class 2 to network-qos 26
platform sand qos map traffic-class 3 to network-qos 32
platform sand qos map traffic-class 4 to network-qos 40
platform sand qos map traffic-class 5 to network-qos 46
platform sand qos map traffic-class 6 to network-qos 48
platform sand qos map traffic-class 7 to network-qos 56
```